### PR TITLE
Release 2.1.10

### PR DESCRIPTION
**Pull request by jakirkham**
```
2016-08-02 1.21.10:
-------------------

Bug fixes:
----------

* Compile files ending with .py, not py.  #1163
* Move root logger to entry points, to not interfere with conda #1164 #1166
* Use setuptools entry points, rather than pre-defined scripts #1165
* Always use the long build prefix to avoid confusion #1168

Contributors:
-------------

* @mingwandroid
* @msarahan
```

### CODE REVIEW

**Comment by jakirkham** [comment]
Could you please take a look @msarahan?

File: recipe/build.sh [hunk 1]
```diff
@@ -3,3 +3,5 @@
 python setup.py install --single-version-externally-managed --record=record.txt
 
 cp bdist_conda.py "${PREFIX}/lib/python${PY_VER}/distutils/command/"
+
+rm -f "${PREFIX}/bin/conda"
```
Comment (msarahan) [inline recipe/build.sh]: This certainly doesn't hurt, and I understand why you're doing it, but we should cut it at some point in the future.

**Comment by msarahan** [comment]
Thanks for taking care of this, @jakirkham

File: recipe/build.sh (hunk 1 above)
Comment (jakirkham) [inline recipe/build.sh]: Yep, probably after we have built the new `conda-build`.

**Comment by jakirkham** [comment]
np. Thanks for taking a look.

**Pull request merged**